MetasysServices/Alarms/Alarm.cs

```diff
@@ -115,10 +115,12 @@ public override bool Equals(object obj)
             {
                 var o = (Alarm)obj;
                 // Compare each properties one by one for better performance
-                return this.Id == o.Id && this.IsAcknowledged == o.IsAcknowledged && this.IsAckRequired == o.IsAckRequired && this.IsDiscarded == o.IsDiscarded
-                       && this.ItemReference == o.ItemReference && this.Message == o.Message && this.Name == o.Name && this.ObjectUrl == o.ObjectUrl &&
-                       this.Priority == o.Priority && this.Self == o.Self && this.TypeUrl == o.TypeUrl
-                       && this.AnnotationsUrl==o.AnnotationsUrl && this.CategoryUrl==o.CategoryUrl && this.CreationTime==o.CreationTime;
+                return this.Id == o.Id && this.IsAcknowledged == o.IsAcknowledged && this.IsAckRequired == o.IsAckRequired
+                    && this.IsDiscarded == o.IsDiscarded && this.ItemReference == o.ItemReference && this.Message == o.Message
+                    && this.Name == o.Name && this.ObjectUrl == o.ObjectUrl && this.Priority == o.Priority
+                    && this.Self == o.Self && this.TypeUrl == o.TypeUrl && this.Type == o.Type && this.AnnotationsUrl == o.AnnotationsUrl
+                    && this.CategoryUrl == o.CategoryUrl && this.Category == o.Category && this.CreationTime == o.CreationTime
+                    && this.TriggerValue == o.TriggerValue;
```

Comment:
Please add the method 'Equals' to the class Measurements as well and use it instead of '=='  (otherwise the tests on the Alarm object will fail).
Also be sure that all the tests passed successfully.

Reply:
Test Cases are handled in another PR [#108](https://github.com/metasys-server/basic-services-dotnet/pull/108)